src/exchanges/oasis/OasisExchangeService.js

```diff
@@ -34,6 +34,9 @@ export default class OasisExchangeService extends PrivateService {
     const buyToken = this.get('token').getToken(currency);
     const daiAmountEVM = daiValueForContract(amount);
     const minFillAmountEVM = daiValueForContract(minFillAmount);
+    const nonce = await this.get('transactionManager')
```

Comment:
I actually tried it this way first and it wasn't working for some reason. I'll try it again, but I also created a separate issue for reworking these to use `formatHybridTx` (which will cover this either way).

Reply:
After working with this a bit more, I think it will be easier to fix this by passing it through `format` on the new issue. As it is now, removing `{ hybrid: false }` causes it to call `createHybridTx` more than once, which then causes problems with nonce generation (I think this is because both `build` and `buyAllAmount` go through the proxy). Untangling it seems complex enough that we might as well reorganize it in the process